### ISSUE TITLE
Cannot find module node_modules/lodash-es/get

### ISSUE THREAD

**Issue by dlutton** [bknd-io/bknd#17]
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/node_modules/lodash-es/get' imported from /node_modules/bknd/dist/ui/chunks/chunk-ULQ54IAG.js
Did you mean to import "lodash-es/get.js"?
    at finalizeResolution (node:internal/modules/esm/resolve:265:11)
    at moduleResolve (node:internal/modules/esm/resolve:933:10)
    at defaultResolve (node:internal/modules/esm/resolve:1157:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:227:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:87:39)
    at link (node:internal/modules/esm/module_job:86:36)

I get this error locally when I try hitting the /admin route in the browser.

**Comment by dswbx** [comment]
hey @dlutton thanks for reporting, sorry that you experience this issue! Could you tell me which environment/adapter you are using? This helps me trying to reproduce the issue. I can see that you're embedding the Admin component and using ESM environment, so maybe astro or remix?

**Comment by dswbx** [comment]
@dlutton Ah yes you're probably using astro, I could reproduce the issue. While I'm checking how to fix it, you could adjust your `astro.config.mjs` file to look like the following:

```mjs
// @ts-check
import { defineConfig } from "astro/config";

import react from "@astrojs/react";

// https://astro.build/config
export default defineConfig({
   integrations: [react()],
   vite: {
      optimizeDeps: {
         exclude: ["astro:db"],
         include: ["lodash-es"],
      },
      resolve: {
         dedupe: ["react", "react-dom"],
      },
      ssr: {
         external: ["react", "react-dom"],
         noExternal: ["bknd", "lodash-es"],
      },
   },
});
```

This will make it work as a workaround. But I will try to fix it asap so that this isn't necessary anymore!

**Comment by dswbx** [comment]
@dlutton the issue should be resolved now in version 0.2.2

/cc @cameronapak: I think you also had this issue (borrowed your fixing astro configuration for the workaround)
